Project.toml

```diff
@@ -1,14 +1,3 @@
-name = "Ectotherm"
-uuid = "b296eefe-8ffd-498e-9f4f-4dd5a85c833d"
-authors = ["BiophysicalEcology"]
-version = "0.1.0"
-
-[compat]
+[deps]
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
-julia = "1"
-
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Why remove these? Its needed for defining any tests.

Reply:
I have put them back now